components-compose/src/main/java/uk/gov/hmrc/components/compose/ui/theme/Type.kt

```diff
@@ -41,3 +41,36 @@ val HmrcTypography = Typography(
     )
     */
 )
+
+object CustomHmrcTypography {
+    val h3 = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Bold,
+        fontSize = 48.sp
+    )
+    val h4 = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Bold,
+        fontSize = 30.sp
+    )
+    val h5 = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Bold,
+        fontSize = 20.sp
+    )
+    val h6 = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Bold,
+        fontSize = 16.sp
+    )
+    val info = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Normal,
+        fontSize = 16.sp
+    )
```

Comment:
Please can we add the colour here as otherwise `info` text has the same definition as body. The `info` text should always have color `grey1`.